firebase-common/src/main/java/com/google/firebase/FirebaseApp.java

```diff
@@ -472,7 +472,7 @@ public boolean isDefaultApp() {
   /** @hide */
   @VisibleForTesting
   @RestrictTo(Scope.TESTS)
-  void initializeAllComponents() {
+  public void initializeAllComponents() {
```

Comment:
Instead of making it public, I suggest creating a function in the benchmark app under `com.google.firebase` like:

```kotlin

internal fun initializeAllComponentsForBenchmark(app : Firebase) {
  app.initializeAllComponents()
}
```

This will overcome the visibility restriction and also prevent `app. initializeAllComponents()` from showing up in autocomplete for 3p developers.

Reply:
Done